checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -299,6 +302,24 @@ protected AnnotatedTypeFormatter createAnnotatedTypeFormatter() {
                 super.methodFromUse(tree);
         AnnotatedExecutableType method = mfuPair.first;
 
+        // Change the type any uninferred type arguments to @NonNull @Initialized.
+        SimpleAnnotatedTypeScanner<Void, Void> scan =
+                new SimpleAnnotatedTypeScanner<Void, Void>() {
+                    @Override
+                    protected Void defaultAction(AnnotatedTypeMirror type, Void aVoid) {
+                        if (type == null) {
+                            return null;
+                        }
+                        if (type.getKind() == TypeKind.WILDCARD
+                                && ((AnnotatedWildcardType) type).isUninferredTypeArgument()
+                                && ignoreUninferredTypeArguments) {
+                            type.replaceAnnotations(qualHierarchy.getBottomAnnotations());
```

Comment:
This is not working well for me and it's also not clear why this should be a Nullness Checker specific change.
It works better if instead of adding all bottom annotations, this adds only NONNULL.
In general, I would try using the same defaults as `java.lang.Object` would get. There is no easy way to do this, I think, other than asking the type factory to annotate `java.lang.Object` and then taking the annotations from there.